types/unist-util-remove-position-test.ts

```diff
@@ -0,0 +1,18 @@
+import {Node} from 'unist'
+
+import remark = require('remark')
+
+import removePosition = require('unist-util-remove-position')
+
+const tree: Node = remark().parse(
+  'Some _emphasis_, **importance**, and `code`.'
+)
+
```

Comment:
Could we add an additional test checking for use as a plugin?

```suggestion

remark()
  .use(() => (tree) => removePosition(tree))
  .process('Some _emphasis_, **importance**, and `code`.')
```